src/lib/y2storage/fake_device_factory.rb

```diff
@@ -566,7 +556,7 @@ def create_lvm_lv(parent, args)
       size = args["size"] || DiskSize.zero
       raise ArgumentError, "\"size\" missing for lvm_lv #{lv_name}" unless args.key?("size")
 
-      lvm_lv = parent.create_lvm_lv(lv_name, size.size)
+      lvm_lv = parent.create_lvm_lv(lv_name, size)
```

Comment:
No, this code is already using the Ruby wrapper, which means we operate directly with `Y2Storage::DiskSize` objects. So we always consistently use `DiskSize` objects instead of the old mixture of `Fixnum` for `Storage` and `DiskSize` for "high level" code.

Note: `size.to_i` would also work in this particular case, but there is no need to convert disk sizes to bytes any more.